### PR TITLE
NW6 | Sabella Fisseha | Feature active row

### PR DESCRIPTION
**Pull request by Sabella-8**
## Learners, PR Template

Self checklist

- [ ] I have committed my files one by one, on purpose, and for a reason
- [ ] I have titled my PR with COHORT_NAME | FIRST_NAME LAST_NAME | REPO_NAME | WEEK 
- [ ] I have tested my changes
- [ ] My changes follow the [style guide](https://syllabus.codeyourfuture.io/guides/code-style-guide/)
- [ ] My changes meet the [requirements](./README.md) of this task

## Changelist

Briefly explain your PR.

## Questions

Ask any questions you have for your reviewer.

### CODE REVIEW

File: src/components/SearchResult/TableBody.jsx [hunk 1]
```diff
@@ -1,6 +1,11 @@
-// import FakeBookings from "@/data/fakeBookings.json";
-
+import FakeBookings from "@/data/fakeBookings.json";
+import React from "react";
+import { useState } from "react";
 function TableBody(props) {
+  const [selectedRow, setSelectedRow] = useState("unselect");
+  function handleSelect() {
+    setSelectedRow(selectedRow === "unselect" ? "selected" : "unselect");
+  }
```
Comment (areebsattar) [inline src/components/SearchResult/TableBody.jsx:8]: Good use of state!

File: src/components/Card/Card.scss [hunk 2]
```diff
@@ -1,16 +1,27 @@
...
+  text-decoration: none;
+  position: relative;
+  top: -40%;
+  text-align: center;
+}
+.cardhref:hover {
+  background-color: skyblue;
+  opacity: 0.7;
+  color: black;
 }
```
Comment (areebsattar) [inline src/components/Card/Card.scss:27]: This looks quite better now. Thank you

**Review by areebsattar — APPROVED** [review]

File: src/components/Card/Card.jsx [hunk 3]
```diff
@@ -8,9 +8,10 @@ const Card = (props) => {
       <h2>{title}</h2>
       <img src={image} alt={title} />
 
-      <a href={url}> {title}</a>
-
-
+      <a href={url} className="cardhref ">
+        {" "}
```
Comment (Ara225) [inline src/components/Card/Card.jsx:12]: This line isn't needed a plain old space should do the trick, or if you need more than one space, you can use &nbsp; which forces HTML to insert a space (normally if you have more than one space next to eachother, HTML only renders one

File: src/components/Card/Card.scss [hunk 4]
```diff
@@ -1,16 +1,27 @@
...
+  color: rgb(255, 255, 255);
+  text-decoration: none;
+  position: relative;
+  top: -40%;
+  text-align: center;
+}
+.cardhref:hover {
+  background-color: skyblue;
+  opacity: 0.7;
+  color: black;
```
Comment (Ara225) [inline src/components/Card/Card.scss:26]: I like the hover styles, always a nice touch to have some

**Pull request merged**